app/src/main/java/org/mozilla/focus/session/SessionCallbackProxy.java

```diff
@@ -24,7 +27,7 @@ public SessionCallbackProxy(Session session, IWebView.Callback delegate) {
     @Override
     public void onPageStarted(@NonNull String url) {
         session.setLoading(true);
-        session.setSecure(false);
+        session.setSecurityInfo(new Session.SecurityInfo(false, "", ""));
```

Comment:
No change necessary but I wonder what the performance effects of allocating here (and in onFinished is). Probably nothing because WebView (both Java and JS) probably does much worse than us :).

Reply:
Yeah, probably negligible. onPageStarted() / onPageFinished() doesn't happen that often. We _could_ create constants as we are ignoring the `host` and `issuer` parts here.